bundler/spec/bundler/dependency_spec.rb

```diff
@@ -170,5 +170,15 @@
     it "includes all platforms" do
       expect(subject).to eq(platforms.merge(deprecated))
     end
+
+    it "is on the current platform" do
+      engine = Gem.win_platform? ? "windows" : RUBY_ENGINE
+
+      dep = described_class.new(
+        "test_gem",
+        "1.0.0",
+        { "platforms" => "#{engine}_#{RbConfig::CONFIG["MAJOR"]}#{RbConfig::CONFIG["MINOR"]}" },
+      )
+    end
```

Comment:
Maybe some rebasing issue, but this commit is missing the actual check that the dependency is in the current platform as described by the spec. It's added in a later commit, but reworking git history may help future spelunkers.

Reply:
Good catch. Yeah a rebasing issue. It's corrected